requirements-dev.txt

```diff
@@ -1,5 +1,5 @@
-flake8
-flake8-quotes
+flake8 >= 6.0.0, < 7.0
```

Comment:
would that work: `flake8 ~= 6.0`, `flake8-quotes~=3.0` ?

Reply:
updated